### ISSUE TITLE
Vue.js 3 - problem jquery - does not provide an export named 'default'

### ISSUE THREAD

**Issue by luandiego7** [godbasin/vue-select2#58]
Hello guys.
I'm facing this problem with jquery plugin when i try use this select2.

![Captura de Tela 2022-05-02 às 20 48 43](https://user-images.githubusercontent.com/30122229/166343679-03eaa155-a7e0-4be3-a594-d2ffa8629b2e.png)

![Captura de Tela 2022-05-02 às 20 47 59](https://user-images.githubusercontent.com/30122229/166343629-3a4c0748-d11e-40bd-b803-35a05f25a6b4.png)

My dependencies:

![Captura de Tela 2022-05-02 às 20 51 04](https://user-images.githubusercontent.com/30122229/166343861-cca0e001-3de5-48dd-be9b-5212ff01ee3f.png)

**Comment by vivazzi** [comment]
I was able to find a solution to the bug and I created pull request https://github.com/godbasin/vue-select2/pull/61. In comment I described about the bug more detail

**Comment by vivazzi** [comment]
Unfortunately, my pull request https://github.com/godbasin/vue-select2/pull/61 does not completely solve this problem. More details: https://github.com/godbasin/vue-select2/pull/61#issuecomment-1139314582

**Comment by jacoherotel** [comment]
same here

**Comment by mimbar** [comment]
i add this in vite.config.js. [Reference](https://github.com/vitejs/vite/issues/813)

```
optimizeDeps: {
        include: ['linked-dep', 'Select2', 'jquery']
},
```

but it lead to another problem
<img width="611" alt="image" src="https://user-images.githubusercontent.com/2446798/188305085-fd54d9bf-d3cb-4466-b20f-30a86f9e609d.png">

anyone?

**Comment by vivazzi** [comment]
@mimbar I think we need to refuse of jQuery and its plugins, because Vue has all that we need to create different plugins. For example, for select there is vue-select, https://github.com/sagalbot/vue-select

**Comment by mimbar** [comment]
@vivazzi that's right. after considering the impact, i chose to move on from jquery and used everything that built using vue.
